core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -233,15 +365,16 @@ import akka.util.UUIDComparator
       awaitingFlush match {
         case Some(replyTo) =>
           log.debug("External flush request")
-          if (sortedBuffer.nonEmpty) {
-            // TODO break into batches
-            write(sortedBuffer, Vector.empty[(Serialized, TagPidSequenceNr)], tagPidSequenceNrs, awaitingFlush)
+          if (buffer.pending.nonEmpty) {
+            // TODO break into batches - check, but this is now done as buffer always batches
```

Comment:
still TODO?

Reply:
The places we didn't batch correctly are now both fixed as part of this PR